source/tools/testConsistency/android-renderer/AndroidDriver/app/src/androidTest/assets/README.md

```diff
@@ -0,0 +1 @@
+Add the samples directory here to run the tests
```

Comment:
Do we actually need this readme or can we have a top-level readme instead?

Reply:
Also why assets + resources both? I see samples appear in the resources folder and not much in the assets folder? 

---
In reply to: [471243745](https://github.com/microsoft/AdaptiveCards/pull/4326#discussion_r471243745) [](ancestors = 471243745)